packages/ketcher-polymer-editor-react/src/components/monomerLibrary/monomerLibraryGroup/usePreview.ts

```diff
@@ -0,0 +1,61 @@
+/****************************************************************************
+ * Copyright 2021 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ***************************************************************************/
+import { RefObject, useState } from 'react'
+import { MonomerItemType } from '../monomerLibraryItem/types'
+import { preview } from './consts'
+import { PreviewStyle } from './types'
+
+export const usePreview = (
```

Comment:
I guess, it is worth to move this hook to `hooks` folder. I bet, this won't be the only case of using it.